samples/AzureKeyVault/apiProperties.json

```diff
@@ -31,6 +31,7 @@
           "properties": {
             "IsFirstParty": "False"
           },
+          "redirectMode": "Global",
           "redirectUrl": "https://global.consent.azure-apim.net/redirect",
```

Comment:
Do we need to specify both 'redirectMode' and 'redirectUrl'?

Reply:
Yes. Note the same with Facebook and O365 connector in this repo.